### DISCUSSION TITLE
How to use Handlebars with parceljs ?

### DISCUSSION THREAD

**Discussion by strukturart** [handlebars-lang/handlebars.js#1854] [Q&A]

**Comment by strukturart** [comment]
`import Handlebars from "handlebars";
`

`@parcel/core: No transformers found for application/index.x-handlebars-template.
`

**Reply by jaylinski** [reply]
Can you give us a complete example of your app?

**Comment by strukturart** [comment]
I want to create a sample messenger app for kaiOs

https://github.com/strukturart/watermelon

**Reply by jaylinski** [reply]
Before I try to debug your issue: are you sure you want to use handlebars? There are more modern libraries for rendering HTML. Like Vuejs, React, or lit-html.

**Reply by strukturart** [reply]
my app should only be a prototype for the ui/ux app with no real data. kaiOs 2.x is also very outdated because it uses ff48.

**Reply by strukturart** [reply]
do you think it is possible to use handlebars with parcel ?

**Reply by strukturart** [reply]
now it works the script type was wrong.

```JS
    <script id="template-chat" type="x-handlebars-template">
            {{#each data}}
                {{#DocumentList}}
                <article tabindex="{{@index}}">
                    <div class="user-date"><span>{{user}}</span> {{transform_date time}}</div>
                        <div>{{message}}</div>
                </article>
                {{/DocumentList}}
            {{/each}}
        </script>
```
